.github/workflows/update-kni-tutorial.yml

```diff
@@ -0,0 +1,46 @@
+---
+name: Update KNI-tutorial repository
+on:
+  workflow_dispatch:
+jobs:
+  update-repo:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write  # To push a branch 
+    steps:
+      - name: Checkout khiops
+        uses: actions/checkout@v3
+        with:
+          path: khiops
+      - name: Get Khiops version
+        run: |
+          KHIOPS_VERSION=$(grep "KHIOPS_VERSION" khiops/src/Learning/KWUtils/KWKhiopsVersion.h | cut -d"(" -f2 | cut -d")" -f1)
+          echo "KHIOPS_VERSION=${KHIOPS_VERSION}" >> "$GITHUB_ENV"
+      - name: Checkout KNI-tutorial
+        uses: actions/checkout@v3
+        with:
+          repository: KhiopsML/KNI-tutorial
+          ssh-key: ${{ secrets.KNI_TUTORIAL_SSH_KEY }}
+          path: tutorial
+      # Copy sources files and README.md from khiops repository to KNI-tutorial
+      # README.md and *.c files are generated by cmake configure
+      - name: Update local KNI-tutorial
+        run: |
+          cmake -B khiops/build -S khiops -DMPI=OFF -DBUILD_JARS=OFF -DBUILD_LEX_YACC=OFF -DTESTING=OFF 
+          cp khiops/build/tmp/kni.README.md  tutorial/README.md
+          cp khiops/build/tmp/KNIRecodeFile.c  tutorial/cpp/
+          cp khiops/build/tmp/KNIRecodeMTFiles.c tutorial/cpp/
+          cp khiops/src/Learning/KhiopsNativeInterface/KhiopsNativeInterface.h tutorial/include/
+          cp khiops/src/Learning/KNITransfer/KNIRecodeFile.h tutorial/cpp/
+          cp khiops/src/Learning/KNITransfer/KNIRecodeMTFiles.h tutorial/cpp/
+          cp packaging/common/KNI/java/* tutorial/java/
+      - name: Update remote KNI-tutorial
+        run: |-
+          cd tutorial
+          git config --global user.email "khiops.bot@no-mail.com"
+          git config --global user.name "khiops bot"
+          git status
```

Comment:
removed

Reply:
I don't see the change

Reply:
The is still `cd tutorial` but I'm sure this line is necessary. For the others I don't know (and I can't test it because the action `Update KNI-tutorial repository` is missing. Perhaps it will be there after the merge)